common/persistence/sql/sqlplugin/sqlite/driver_cgo.go

```diff
@@ -0,0 +1,43 @@
+// The MIT License
```

Comment:
can we split this PR into 2
non cgo related PR (unit test & src)
cgo related PR (we may need to discuss, since cgo probably introduced lots of dependencies?)

i mean these 2 files
`driver_cgo.go`
`driver_nocgo.go`

Reply:
Sure thing. I can default to the modernc dependency for now since that's compatible with both cgo and non-cgo builds.